static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -72,7 +76,7 @@ export default function FeedbackListPage({}: Props) {
             <ErrorBoundary>
               <LayoutGrid>
                 <FeedbackFilters style={{gridArea: 'filters'}} />
-                {hasSetupOneFeedback ? (
+                {hasSetupOneFeedback || isFromSlack ? (
```

Comment:
i feel like we should just look to see if the link has a feedback slug, and if so, always show the feedback

Reply:
that works, i'll change this